### PR TITLE
Remove Ophan Tracking Duplication in DCR Bundle

### PR DESCRIPTION
**Pull request by nitro-marky**
## What does this change?

Currently in DCR ophan ng is imported twice, once by DCR and the second from its inclusion in the DCR commercial bundle. This has potentially led to incorrect or misleading tracking information sent to ophan when viewing articles. 

The ophan/ng module has been aliased to stop it being bundled, and therefore duplicated, in DCR.

Before in DCR (ophan calls have been made by the commercial bundle and the ophan script): 
Screenshot 2020-11-18 at 13 51 21

After in DCR (ophan calls are only made by ophan)
Screenshot 2020-11-18 at 13 53 55

### CODE REVIEW

File: webpack.config.dcr.js [hunk 1]
```diff
@@ -17,8 +17,14 @@ config.entry = {
...
 };
 
+// The Ophan alias removes duplicating the js in the commercial bundle sent to DCR.
 module.exports = webpackMerge.smart(config, {
     output: {
         path: path.join(__dirname, 'static', 'target', 'javascripts'),
     },
+    resolve: {
+        alias: {
+            "ophan/ng": path.join(__dirname, 'static', 'src', 'javascripts', 'projects', 'commercial', 'modules', 'ophan-dcr.js'),
```
Comment (jfsoul) [inline webpack.config.dcr.js:27]: I'm still not really sure where this file should live. Maybe next to `commercial.dcr.js` in `bootstrap` called `commercial-ophan.dcr.js`?

**Review by jfsoul — APPROVED** [review]

File: webpack.config.dcr.js (hunk 1 above)
Comment (nitro-marky) [reply webpack.config.dcr.js:27]: Yeah, that's probably a better way of doing it. I'll make the change.

File: static/src/javascripts/bootstraps/commercial-ophan.dcr.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+// @flow strict
+// Monkey patch to facilitate the removal of ophan tracking from the commercial bundle sent to DCR.
+export default {
+    // $FlowFixMe
+    record: (value) => {
+        if (window && window.guardian && window.guardian.ophan && window.guardian.ophan.record)
+        {
+            window.guardian.ophan.record(value);
+        }
```
Comment (sndrs) [inline static/src/javascripts/bootstraps/commercial-ophan.dcr.js:9]: I think you can do this if you prefer:
```suggestion
        window?.guardian?.ophan?.record?.(value);
```

**Review by sndrs — APPROVED** [review]
suggestion for tersering (?) up the checks if you want, but not a blocker

**Pull request merged**